python/lsst/ts/m2/mock_server.py

```diff
@@ -0,0 +1,924 @@
+# This file is part of ts_m2.
+#
+# Developed for the Vera Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+import socket
+import json
+import asyncio
+import numpy as np
+
+from lsst.ts import salobj, tcpip
```

Comment:
Consider splitting this import statement into two. You are basically importing two different modules here, we usually reserve multiple imports for when you are importing things from the same module (like the one you did bellow `from . import CommandStatus, DetailedState`)

Reply:
Done in the commit: fdeb2fc. Thanks!